tests/mirage/me/crate-owner-invitations/list-test.js

```diff
@@ -5,15 +5,15 @@ import fetch from 'fetch';
 import { setupTest } from '../../../helpers';
 import setupMirage from '../../../helpers/setup-mirage';
 
-module('Mirage | GET /api/v1/me/crate_owner_invitations', function (hooks) {
+module('Mirage | GET /api/private/crate_owner_invitations', function (hooks) {
```

Comment:
we should probably also keep the tests for the old endpoint until we receive it :)

Reply:
Reintroduce it here: [9fde45f](https://github.com/rust-lang/crates.io/pull/5397/commits/9fde45f06005902a17e91869ede891a755b95342)